### ISSUE TITLE
Shelly 2.5 Command Requires Lower Case

### ISSUE THREAD

**Issue by cuhouse** [FlyingDiver/Indigo-Shims#8]
Flying/Diver,

I am adding a Shelly 2.5 Dual Relay switch to Indigo using the Shims plugin.  Shelly has native MQTT client in device and I want to try this vice flashing with Tasmota.
From the Shelly [documentation](https://shelly-api-docs.shelly.cloud/#shelly2-5-mqtt) it states that it requires lower case on/off.
"shellies/shellyswitch25-<deviceid>/relay/<i>/command accepts on, off or toggle"
When pressing the On/Off in the Indigo Device it seems the plugin sends an On/Off (uppercase O)and the device will not respond.  I use MQTT Explorer and can send an on/off (lowercase O) and the device responds appropriately.
When Indigo sends command:

![Screen Shot 2020-01-25 at 10 11 06 AM](https://user-images.githubusercontent.com/54967289/73124540-a07f4500-3f6a-11ea-9544-15654f9aa40c.png)

Current Trigger and Device Settings:

![Screen Shot 2020-01-25 at 11 52 41 AM](https://user-images.githubusercontent.com/54967289/73124459-d3750900-3f69-11ea-9878-d7882591a308.png)

![Screen Shot 2020-01-25 at 11 53 25 AM](https://user-images.githubusercontent.com/54967289/73124478-f1426e00-3f69-11ea-96bc-22f67989c5b8.png)

Thank you and if this is too much effort, please advise and I will pursue replacing the firmware.  I was trying this to see if it might be an option for those folks that don't want to reflash devices.

CUHouse

**Comment by FlyingDiver** [comment]
Try this pre-release:

https://github.com/FlyingDiver/Indigo-Shims/releases/tag/0.1.3

**Comment by cuhouse** [comment]
Thanks for the FAAAST update.
The on/off command from the Indigo device now turns the Shelly 2.5 on and off.  I did a quick check of my Tasmota flashed switches and all remains normal with them too.  I do have a remaining issue with the new Shelly 2.5. I have turned Shims debugging on to determining what is happening.  When I turn the Shelly 2.5 on with Indigo, the status in Indigo device updates to green but then in a couple seconds the green indication turns off.  I see extra triggers firing in the "on" condition so troubleshooting the issue at this time.
Thanks again,
CUHouse
